DebuggerGame/Assets/Scripts/Save Scripts/SaveState.cs

```diff
@@ -5,37 +5,49 @@
 using UnityEditor;
 using UnityEngine;
 
-[CreateAssetMenu(fileName = "New Inventory", menuName = "Inventory System/Inventory")]
-public class InventorySystem : ScriptableObject, ISerializationCallbackReceiver
+[CreateAssetMenu(fileName = "New Save State", menuName = "Save System/Save State")]
+public class SaveState : ScriptableObject, ISerializationCallbackReceiver
 {
     public string savePath;
-    public ArthropodDatabase database;
-    public List<InventorySlot> Container = new List<InventorySlot>();
 
+    [SerializeField]
+    //Level info
+    public int currentLevel;
```

Comment:
Could this save data be a struct so that the save data is not defined independently in SaveManager and SaveState?

Reply:
I was gonna work on the persistent more after merging first and creating a new branch